assets/Zed/js/modules/form-toggler.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+'use strict';
+
+$(document).ready(function() {
+    $('.dependent-trigger').each(function (index, selectItem) {
+        $('.dependent-child').parent('.form-group').addClass('dependent-parent');
+        var groupName = $(selectItem).attr('data-dependent-group');
+        var visibleTarget = getTargetElements(groupName, selectItem.value);
```

Comment:
Why the name is `visibleTarget`? Where is the check on visibility?

Reply:
Renamed.